jenkins/jenkins_home/config.xml

```diff
@@ -87,6 +87,7 @@
         <string>instance-event</string>
         <string>instance-events-cron</string>
         <string>pipeline3</string>
+        <string>reset-instance</string>
```

Comment:
@chinzou , are we creating a new job "Reset Instance" to reset an instance?
If so, it appears that the actual job files are missing from the PR?